src/Engine/Component/GeometryComponent.cpp

```diff
@@ -73,33 +73,32 @@ void TriangleMeshComponent::generateTriangleMesh( const Ra::Core::Asset::Geometr
     std::string name( m_name );
     name.append( "_" + m_contentName );
 
-    std::string meshName = name;
-    meshName.append( "_Mesh" );
+    std::string meshName = m_contentName;
+    // meshName.append( "_Mesh" );
 
     m_displayMesh = Ra::Core::make_shared<Mesh>( meshName );
 
     Ra::Core::Geometry::TriangleMesh mesh;
     Ra::Core::Geometry::TriangleMesh::PointAttribHandle::Container vertices;
     Ra::Core::Geometry::TriangleMesh::NormalAttribHandle::Container normals;
-
-    const auto T = data->getFrame();
-    const Ra::Core::Transform N( ( T.matrix() ).inverse().transpose() );
-
+    // data->getFrame()
     vertices.resize( data->getVerticesSize(), Ra::Core::Vector3::Zero() );
 
+    // Use std::copy ?
```

Comment:
Yes, please update this code with algorithms from the `std` library, like copy and transform.

Reply:
The replaced code was equivalent of a std::transform. Here, we have to copy the data without modification. std::copy will do that but parallel execution policy are not yet well supported on several systems. Well, this code being used once, at the creation of a component, if its execution is sequential instead of parallel, the time penalty will not be so much. So, OK, I'll change this, trying the std::par policy.

Reply:
STD::transform can be used for the normals.